frontend/mr-admin-flate/src/components/tiltakstyper/TiltakstypeRad.tsx

```diff
@@ -16,24 +16,19 @@ export function TiltakstypeRad({ tiltakstype }: Props) {
       classname={styles.listerad_tiltakstype}
       testId="tiltakstyperad"
     >
-      <BodyShort
-        aria-label={`Navn på tiltakstype: ${tiltakstype.navn}`}
-        size="medium"
-      >
+      <BodyShort aria-label={`Navn på tiltakstype: ${tiltakstype.navn}`}>
         {tiltakstype.navn}
       </BodyShort>
-      <BodyShort size="medium">
+      <BodyShort>
```

Comment:
Aria-label her også, så skjermleserbruker forstår konteksen av hva de har navigert til

Reply:
Hvis det er til tiltakstypestatusen du mener så ligger det en aria-label på selve statusen

`export function Tiltakstypestatus({ tiltakstype }: Props) {
  const status = tiltakstype.status;

  return (
    <Tag
      aria-label={`Status for tiltakstype: ${status}`}
      variant={
        status === "Aktiv"
          ? "success"
          : status === "Planlagt"
          ? "info"
          : "neutral"
      }
    >
      {status}
    </Tag>
  );
}`

Reply:
Hvis det er tiltakstypestatusen du mener så har den en egen aria-label i komponenten
![image](https://user-images.githubusercontent.com/21334782/227141641-3e545766-da03-4968-93b6-258a014f9024.png)